src/flb_input_chunk.c

```diff
@@ -1363,8 +1379,10 @@ int flb_input_chunk_append_raw(struct flb_input_instance *in,
         diff = 0;
     }
 
-    if (diff != 0) {
-        flb_input_chunk_update_output_instances(ic, diff);
+    if (real_diff != 0) {
+        flb_debug("[input chunk] update output instances with new chunk size diff=%d",
+                  real_diff);
```

Comment:
Could you add the chunk name to the debug message? If the similar issue happened in the future, we could have some `audit` logs on the change to the chunk size update.

Reply:
I added new debug messages which track the life cycle of fs_chunks_size which add the chunk name to the relevant debug messages.